### ISSUE TITLE
Joomla 4 RC5 - Target Position  unclear description

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-cms#35025]
### What needs to be fixed
There is a setting named Target Position in the menu module and it's absolutely not clear what it does. Maybe someone could rewrite to the description below where Popups could appear. 

![grafik](https://user-images.githubusercontent.com/828371/127851034-383709e8-2ab7-49d4-82b0-3e0e000163d8.png)

### Why this should be fixed
to avoid confusion

### How would you fix it
Someone could rewrite the description to explain what it does

**Comment by brianteeman** [comment]
This is exactly the same as in Joomla 3 except that the description is not hidden.

I have read and read that description multiple times now and I cant see how it could be any clearer

JavaScript values to **position a popup window,** eg top=50, left=50, width=200, height=300.

**Comment by ChristineWk** [comment]
I have also read and read - and don't know :-)
Maybe: To position JavaScript values of a popup window, e.g. ...<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35025">issues.joomla.org/tracker/joomla-cms/35025</a>.</sub>

**Comment by coolcat-creations** [comment]
Ok, fun - never knew there is a popup window anywhere - what Popup window?

**Comment by brianteeman** [comment]
Try it and see. It might be so old that it doesnt do anything at all

**Comment by brianteeman** [comment]
If the menu type is URL **and** the target is **New Window without navigation** then the settings here determine the size of the new window

**Comment by coolcat-creations** [comment]
Thank you! So its the setting for a new window and not for a popup? Maybe we could write it like that in the description?
Or rename the setting in the Menu to Popup instead of New Window?

**Comment by brianteeman** [comment]
1. we're in language freeze so nothing can be changed right now
2. personally I wouldnt bother to change it in the future. its probably been this way since the stone age

**Comment by chmst** [comment]
Personally I would remove the param. I never tried but can this be accessible?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35025">issues.joomla.org/tracker/joomla-cms/35025</a>.</sub>

**Comment by brianteeman** [comment]
> Personally I would remove the param. I never tried but can this be accessible?
> 

It can be if someone wanted it to.

Too late to remove it now though for 4.0
